routes.go

```diff
@@ -23,119 +23,132 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/justinas/alice"
-	"github.com/prometheus/client_golang/prometheus"
 	"github.com/xmidt-org/argus/store"
-	"github.com/xmidt-org/themis/xhealth"
-	"github.com/xmidt-org/themis/xhttp/xhttpserver"
-	"github.com/xmidt-org/themis/xmetrics"
-	"github.com/xmidt-org/themis/xmetrics/xmetricshttp"
+	"github.com/xmidt-org/arrange"
+	"github.com/xmidt-org/arrange/arrangehttp"
+	"github.com/xmidt-org/candlelight"
+	"github.com/xmidt-org/httpaux"
+	"github.com/xmidt-org/touchstone/touchhttp"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux"
 	"go.uber.org/fx"
 )
 
-type ServerChainIn struct {
+type PrimaryRouterIn struct {
 	fx.In
+	Router    *mux.Router `name:"server_primary"`
+	APIBase   string      `name:"api_base"`
+	AuthChain alice.Chain `name:"auth_chain" optional:"true"`
```

Comment:
Should we make this a required component since we always use it as if it were defined?